holidayextras/ui-brand-hx
# Hovers

#### What does this PR do? (please provide any background)
* Add subtle underline to all article links
* Add similar underlines to all links inside alerts
* Nav-tabs should look like default buttons
* Nav-pills should also look like default buttons
* Table row hover should match default button hover
* navbar-inverse hover colour

#### What tests does this PR have?
N/A

#### How can this be tested?
You can test this by using https://github.com/holidayextras/pattern-library
In the package.json switch the UI-BRAND-HX call to use this branch. If you don't know how, then please ask.

Check out these changes on...
* pattern-library/atoms/articles
* pattern-library/atoms/tables
* pattern-library/molecules/navigation
* pattern-library/molecules/alerts/


#### Screenshots / Screencast
![screen shot 2016-01-07 at 17 14 29](https://cloud.githubusercontent.com/assets/400087/12177052/43f2578c-b562-11e5-992a-0aa38bc90a97.png)

![screen shot 2016-01-07 at 17 14 47](https://cloud.githubusercontent.com/assets/400087/12177057/470428c4-b562-11e5-918a-75fd4ff44ce6.png)

![screen shot 2016-01-07 at 17 15 02](https://cloud.githubusercontent.com/assets/400087/12177060/499630a0-b562-11e5-99e0-ab181b20c397.png)

![screen shot 2016-01-07 at 17 21 02](https://cloud.githubusercontent.com/assets/400087/12177242/11a8e7ea-b563-11e5-8e14-92126c8e9088.png)



#### What gif best describes how you feel about this work?
![](http://i.giphy.com/Ok5qqc6O1YeOY.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
